src/components/experiment-tracking/tooltip/tooltip.scss

```diff
@@ -0,0 +1,103 @@
+@use '../../../styles/variables' as colors;
+
+@mixin fade-in($waitTime) {
+  animation: wait #{$waitTime}, fade-in 800ms #{$waitTime};
```

Comment:
Love that you're using CSS delay instead of a JS timeout. Well done!

Reply:
thank you 😄